requirements.txt

```diff
@@ -11,7 +11,7 @@ gunicorn==19.9.0
 nose==1.3.7
 pinocchio==0.4.2
 factory-boy==2.12.0
-pylint>=2.4.1
+pylint>=1.9.5
```

Comment:
Let's change it back to the pip3 version :)

Reply:
Done so